components/Common/AvatarGroup/index.stories.tsx

```diff
@@ -0,0 +1,61 @@
+import { userNameToAvatarUrl } from '@/util/github';
+import AvatarGroup from './';
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+
+type Story = StoryObj<typeof AvatarGroup>;
+type Meta = MetaObj<typeof AvatarGroup>;
+
+const AvatarList = {
```

Comment:
This is not a component, so no use of titlecase. Call it `defaultProps` or `avatarListProps`